build_tools/docker/context/install_ccache.sh

```diff
@@ -7,16 +7,25 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 CCACHE_VERSION=4.7.4
+machine=$(uname -m)
+if [[ $machine == x86_64 ]]; then
```

Comment:
nit: Same reason as above
```suggestion
ARCH="$(uname -m)"
if [[ "${ARCH}" == x86_64 ]]; then
```

Reply:
Will fix